go/vumitools/utils.py

```diff
@@ -188,13 +188,6 @@ def set_tag(self, tag):
         TaggingMiddleware.add_tag_to_msg(self.message, tag)
         self.tag = TaggingMiddleware.map_msg_to_tag(self.message)
 
-    def get_tag_info(self):
-        if self.tag is None:
-            raise ValueError("No tag to look up metadata for.")
```

Comment:
So now we don't have this `None` check... Is it ok to pass `None` to `get_tag_info` or are we sure we won't do that?